source/options.css

```diff
@@ -1,3 +1,8 @@
+:root {
+	--github-green: #28a745;
+	--github-red: #cb2431;
+}
+
 html {
 	min-width: 550px;
 	overflow-x: hidden;
```

Comment:
This shouldn't be necessary. If it is, it's more likely that some CSS is configured to be wider than it should be (e.g. width: 100% + padding)

Reply:
I'd suggest merging these 2 rules into only:

```css
html:not(.is-edgium) {
	min-width: 550px;
}
```

Reply:
> `overflow-x` shouldn't be necessary. If it is, it's more likely that some CSS is configured to be wider than it should be (e.g. width: 100% + padding)

That's because of negative margins on `hr` from https://github.com/fregante/webext-base-css/blob/5e49453e04d8b97eee3fed983877a773b0df7b76/webext-base.css#L40-L41.

Body has default margin of `8px`, but `hr` has added with `-17px` margin, more than what is on the `body` (from UA styles).

Reply:
Right, that’s why I suggested dropping that code too. It’s already part of base-css and it doesn’t cause this overflow.

Reply:
@fregante RGH is getting `17px` margin horizontally from somewhere. But this extension is not getting that.

![image](https://user-images.githubusercontent.com/37769974/83040456-e48b6f00-a05c-11ea-8f0e-660aa5ec5294.png)

I'm getting `8px` as opposed to `17px` in RGH's case.

![image](https://user-images.githubusercontent.com/37769974/83040598-100e5980-a05d-11ea-8996-217d81d6b765.png)

Reply:
Likely because `chrome_style:true` is missing here: https://github.com/sindresorhus/notifier-for-github/blob/dd684a3a69c9e200c2e1df6a6680026b588fc719/source/manifest.json#L36 while RGH has it https://github.com/sindresorhus/refined-github/blob/ef3cee995f8ba8da5337d8220f8ceed80d43febf/source/manifest.json#L31

If that fixes it I'll update base-css’s readme

Reply:
> Likely because `chrome_style:true` is missing here:

That indeed seems to be the issue.